### PR TITLE
MUMPFL-110 : Fix state country drop downs

### PR DESCRIPTION
**Pull request by timlevett**
+ Country is now preselected as USA for new form
image
+ State is now preselected as WI for new form
image
+ Fix country order (now A-Z)
screenshot - 07272015 - 03 10 22 pm
+ Fix State Order (now A-Z)
screenshot - 07272015 - 03 11 04 pm
+ Clean up empty state ... of state when a country doesn't have state/province/regions... in our database
http://goo.gl/v8mdMX

### CODE REVIEW

**Comment by apetro** [comment]
:+1:

**Comment by jhanstra** [comment]
:+1:

File: my-profile-webapp/src/main/webapp/my-app/lec/partials/address.html [hunk 1]
```diff
@@ -33,22 +33,22 @@
...
         <div class='form-group'>
             <label for="state">State/Province/Region*</label>
             <select id='state' name="state"
-                    class="form-control" 
-                    required 
-                    ng-model='address.state' >
-                    <option ng-repeat='item in states | filter:{country: address.country} | orderBy:value' ng-if='address.country' value='{{item.key}}' ng-attr-selected="{{isSelection(address.state, item.key)}}">{{item.value}}</option>
+                    class="form-control"
+                    ng-model='address.state' ng-disabled='!address.country || (states | filter:{country: address.country}).length === 0'>
+                    <option  ng-if='!address.country' value=''>Select Country First</option>
```
Comment (apetro) [inline my-profile-webapp/src/main/webapp/my-app/lec/partials/address.html]: Maybe sentence-case as "Select country first".

**Comment by vertein** [comment]
:+1: Does it show the correct country/state when someone goes in to edit mode?

**Comment by timlevett** [comment]
@vertein - yes, yes it does

**Pull request merged**